app/internal/windows/windows.go

```diff
@@ -303,15 +316,31 @@ const (
 	LR_MONOCHROME       = 0x00000001
 	LR_SHARED           = 0x00008000
 	LR_VGACOLOR         = 0x00000080
+
+	FILE_MAP_WRITE = 0x0002
+	FILE_MAP_READ  = 0x0004
+
+	PAGE_READWRITE = 0x00000004
+)
+
+const (
+	GIO_OPEN_URL uintptr = iota + 1 // Custom message
```

Comment:
This belongs in `os_windows.go`, no? Also, I suggest naming it similar to the existing `_WM_WAKEUP`.